{"frames":[{"func":"ggplot2::geom_sf(data = perimetro, fill = NA)","file":"","line_number":0,"end_line_number":0,"character_number":0,"end_character_number":0},{"func":"layer_sf(geom = GeomSf, data = data, mapping = mapping, stat = stat, \n    position = position, show.legend = show.legend, inherit.aes = inherit.aes, \n    params = list(na.rm = na.rm, ...))","file":"","line_number":0,"end_line_number":0,"character_number":0,"end_character_number":0},{"func":"layer(geom = geom, stat = stat, data = data, mapping = mapping, \n    position = position, params = params, inherit.aes = inherit.aes, \n    check.aes = check.aes, check.param = check.param, show.legend = show.legend, \n    layer_class = layer_class)","file":"","line_number":0,"end_line_number":0,"character_number":0,"end_character_number":0},{"func":"fortify(data)","file":"","line_number":0,"end_line_number":0,"character_number":0,"end_character_number":0}],"message":"Error in fortify(data) : object 'perimetro' not found\n"}